docker-compose-dev-mac.yml

```diff
@@ -21,6 +21,7 @@ services:
             SURVEY_RUNNER_URL: http://localhost:5000
             SURVEY_RUNNER_SCHEMA_URL: http://docker.for.mac.host.internal:5000
             SDS_API_BASE_URL: http://docker.for.mac.host.internal:5003
+            CIR_API_BASE_URL: http://host.docker.internal:5004
```

Comment:
Do we want to use `http://docker.for.mac.host.internal` here since it's the mac one?

Reply:
Oops, good spot, yep 👍